src/components/editorForm.tsx

```
@@ -65,7 +65,24 @@ const EditorForm = () => {
 		Object.entries(importMap).forEach(([key, value]) =>
 			currentWorker.addFile(`inmemory://model/node_modules/${key}`, value),
 		)
+		return true
 	}
+
+	useEffect(() => {
+		async function initialAdd() {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: addPackage

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

This dependency is not specified in the hook dependency list.

Either include them or remove the dependency array

(lint/correctness/useExhaustiveDependencies)